server/services/licenceService.js

```diff
@@ -8,7 +8,7 @@ const {
 
 module.exports = function createLicenceService(licenceClient, establishmentsClient) {
 
-    async function getLicence(nomisId) {
+    async function getLicence(nomisId, {populateConditions = false} = {}) {
```

Comment:
what's the value of defaulting `populateConditions` to false.. if it doesn't exist it will be a falsy.. I think you can remove the default value here?